src/logger/index.js

```diff
@@ -38,6 +40,10 @@ const localLoggingFormat = printf( output => {
 // Logging format for production
 const prodLoggingFormat = printf( output => {
 	const { timestamp: time, app, app_type: type } = output;
+
+	// Can't include the timestamp in the JSON
+	delete output.timestamp;
+
 	return `${ time } ${ app }${ type } ${ JSON.stringify( output ) }`;
```

Comment:
given we removed the leading `:`, I think we need to add it in the logging format here (both prod and local). Right?

The current behavior is, for domain `vip:go:server` it shows `vipgo:server` in the logs

Reply:
Oh good catch :)